twoliter/src/project.rs

```diff
@@ -136,22 +136,6 @@ impl Project {
         self.sdk.clone()
     }
 
-    pub(crate) fn sdk(&self) -> Result<Option<ImageUri>> {
```

Comment:
Is this never being read from the Project by anything? What happens if I specify the SDK in the Project? Nothing? Is it ignored?

Does the resolve function during `twoliter update` throw an error if the SDK listed in the project does not match one of the external kits?

What if there are no external kits at all (e.g. in the bottlerocket-core-kit build)? In that case the SDK must come from the project.

Reply:
Yes, the lock resolution verifies both if a sdk exists in Project or auto resolves if not and if there is a mismatch anywhere it will error out